src/raftstore/store/worker/apply.rs

```diff
@@ -1424,10 +1434,11 @@ pub struct Runner {
     host: Arc<CoprocessorHost>,
     delegates: HashMap<u64, ApplyDelegate>,
     notifier: Sender<TaskRes>,
+    sync_log: bool,
```

Comment:
remove sync log here, here we just handle flush wal, no need to care sync log.

Reply:
sysbench test show no performance drop when enable sync log,  also it's more safe to enable this,  so I think it's better enable this.

Reply:
no, one thing in one PR, we don't need sync the data here, remove it.

Reply:
btw, do you test this in raw put case?